OpenEdXMobile/src/main/java/org/edx/mobile/view/WebViewFindCoursesActivity.java

```diff
@@ -19,6 +19,11 @@
 
 import roboguice.inject.ContentView;
 
+/**
+ * @deprecated As of release v2.13, see {@link MainDashboardActivity}, {@link MainTabsDashboardFragment}
+ * and {@link WebViewDiscoverCoursesFragment} as an alternate.
+ */
+@Deprecated
 @ContentView(R.layout.activity_find_courses)
 public class WebViewFindCoursesActivity extends BaseWebViewFindCoursesActivity {
```

Comment:
Not really sure if this should be deprecated as it involves moving the whole workflow of Webview based Find Courses to Fragment.
Shouldn't we do that as part of a separate ticket? Thoughts?

Reply:
Nope! Its only showing the Discovery screen i.e. Courses can be found using it .. but enrollment is totally a different ballgame.
Which needs to be done separately imo.

Reply:
There's a separate ticket to take care of this issue:
https://openedx.atlassian.net/browse/LEARNER-3842